lib/turn.rb

```diff
@@ -0,0 +1,66 @@
+class Turn
+  attr_reader :player1, :player2, :spoils_of_war
+
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = []
+  end
+
+  def type
+    if player1.deck.cards.length < 3 && (player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0))
+      :player1_lost
+    elsif player2.deck.cards.length < 3 && (player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0))
+      :player2_lost
+    elsif player1.deck.rank_of_card_at(0) != player2.deck.rank_of_card_at(0)
+      :basic
+    elsif (player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0)) &&
+      (player1.deck.rank_of_card_at(2) == player2.deck.rank_of_card_at(2))
+      :mutually_assured_destruction
+    elsif player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0)
+      :war
+    end
+  end
+
+  def pile_cards
+    if type == :player1_lost
+      spoils_of_war.concat(player1.deck.cards)
```

Comment:
Ideally we should be able to use our `award_spoils` method instead of manually concatenating here (and on line 30) .